route/build/src/init_model_data.f90

```diff
@@ -587,7 +588,8 @@ SUBROUTINE init_ntopo(nHRU_out, nRch_out,
     !        --> users can modify the hard-coded parameter "maxUpstreamFile" if desired
     if(tot_upstream > maxUpstreamFile) tot_upstream=0
 
-    call system('rm -f '//trim(ancil_dir)//trim(fname_ntopNew))
+    call shr_sys_system('rm -f '//trim(ancil_dir)//trim(fname_ntopNew), ierr)
```

Comment:
In order to run on all systems the shr_sys_system and shr_sys_flush and shr_infnan_isnan needs to be used from the CESM share code. The one issue with this is to ensure it still works with the standalone build.

Reply:
MAybe this isn't needed. You can probably overwrite the file on the open.

Reply:
We will do the same for the isnan check as well -- just remove it.

Reply:
I commented out the isnan check, and add some discussion about it.